playground/AzureContainerApps/AzureContainerApps.AppHost/api.module.bicep

```diff
@@ -145,11 +145,6 @@ resource api 'Microsoft.App/containerApps@2025-02-02-preview' = {
         minReplicas: 0
       }
     }
-    runtime: {
```

Comment:
I'm not sure how this got added in 2 places - maybe it was a merge between my change and the automated change. But this change is now correct. This section shouldn't be here, but instead under `propert/configuration`, which it is above.

Reply:
This is why it needs to be updated in a single PR.